gnucash/import-export/import-main-matcher.c

```diff
@@ -177,6 +177,18 @@ void gnc_gen_trans_list_delete (GNCImportMainMatcher *info)
     g_free (info);
 }
 
+gboolean gnc_gen_trans_list_empty(GNCImportMainMatcher *info)
+{
+    GtkTreeModel *model;
+    GtkTreeIter iter;
+    GNCImportTransInfo *trans_info;
+    
+    g_assert (info);
+    
```

Comment:
We want to remove whitespace wherever possible

Reply:
Sure. I'm only replicating code that I see around, in the hope that I replicate coding standards. See on_matcher_ok_clicked() just below, which I did not touch.